hazelcast/src/main/java/com/hazelcast/internal/config/override/ConfigNodeElementAdapter.java

```diff
@@ -0,0 +1,382 @@
+package com.hazelcast.internal.config.override;
+
+import org.w3c.dom.Attr;
+import org.w3c.dom.DOMException;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NamedNodeMap;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.w3c.dom.TypeInfo;
+import org.w3c.dom.UserDataHandler;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+class ConfigNodeElementAdapter implements Element {
```

Comment:
Consider changing the name, two options:
1. Name it the same the same as in the `*.yaml.*` package: `ElementAdapter`.
2. Rename it to `ConfigOverrideElementAdapter` (and maybe rename also `ElementAdapter` to `YamlElementAdapter`).

Because it gets congusing imo.